packages/moonstone/Scroller/Scroller.js

```diff
@@ -298,49 +298,55 @@ class ScrollerBase extends Component {
 		}
 	}
 
-	getNextEndPoint = (direction, oSpotBounds) => {
-		const bounds = this.uiRefCurrent.getScrollBounds();
-
+	getNextEndPoint = (direction, scrollerBounds, focusedItemBounds, pageDiff) => {
```

Comment:
I prefer a boolean `ignoreCurrentPage` or something than a number `pageDiff` for limitation for clearer understanding.

Reply:
I replaced `pageDiff` with `isInNextPage`.